fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rule/mv/IsNoCallChildrenValidator.java

```diff
@@ -43,11 +87,18 @@ public Boolean visitCall(CallOperator operator, Void context) {
             return false;
         }
 
-        for (int i = 0; i < operator.getChildren().size(); i++) {
-            if (!operator.getChild(i).accept(this, null)) {
-                return false;
-            }
+        if (!Utils.extractColumnRef(operator.getChild(0)).stream().allMatch(keyColumns::contains)) {
```

Comment:
These three if branches only handle if function. So I think would better put these code to above if branch or a method, let other people know these three if branches  only handle if function